eth/downloader/queue.go

```diff
@@ -477,9 +477,10 @@ func (q *queue) ReserveReceipts(p *peerConnection, count int) (*fetchRequest, bo
 // to access the queue, so they already need a lock anyway.
 //
 // Returns:
-//   item     - the fetchRequest
```

Comment:
revert changes to comments

Reply:
done.

Reply:
still has some whitspace changes. please check and remove.